python/install.sh

```diff
@@ -4,7 +4,7 @@ then
 else 
     # Install python 3.10.11 using pyenv, in case it's not already installed
     eval "$(pyenv init -)"
-    PYTHON_VERSION=3.10.11
+    PYTHON_VERSION=3.11
```

Comment:
@claudio-souza-mbh: note to self. After we merge this to master, test if this will update the existing version.